main.js

```diff
@@ -0,0 +1,29 @@
+const arrows = document.getElementById('arrows');
+const container = document.getElementById('container');
+
+function moveImg(event) {
+    const { arrow } = event.target.dataset;
+    const scrollStep = 420;
+    const scrollStepMax = 980;
+    const scrollStepMin = 0;
+    const leftArrow = arrows.children[0];
+    const rightArrow = arrows.children[1];

```

Comment:
it is not critical, but will be better to define these variables outside the function(except arrow)